builder/builder.mk

```diff
@@ -759,7 +756,7 @@ release/promote-oss/dev-to-rc:
 		fi ;\
 		printf "$(CYN)==> $(GRN)found version $(BLU)$$dev_version$(GRN) for $(BLU)$$commit$(GRN) in S3...$(END)\n" ;\
 		veroverride=$(patsubst v%,%,$(VERSION)) ; \
-		tag=$$(echo $(patsubst v%,%,$(VERSION)) | tr '+' '-') ; \
+		tag=$$veroverride ; \
```

Comment:
The `tr '+' '-'` isn't still important here? IIRC it was there in the first place because of Docker tags...

Reply:
`goversion` shouldn't ever spit out a version with a `+` in it.

Reply:
👍